packages/teams-js/src/public/call.ts

```diff
@@ -6,14 +6,21 @@ import { errorNotSupportedOnPlatform, FrameContexts } from './constants';
 import { runtime } from './runtime';
 
 export namespace call {
+  /** Modalities that can be associated with a call. */
   export enum CallModalities {
+    /** Indicates that the modality is unknown or undefined. */
     Unknown = 'unknown',
+    /** Indicates that the call includes audio. */
     Audio = 'audio',
+    /** Indicates that the call includes video. */
     Video = 'video',
+    /** Indicates that the call includes video-based screen sharing. */
     VideoBasedScreenSharing = 'videoBasedScreenSharing',
+    /** Indicates that the call includes data sharing or messaging. */
     Data = 'data',
   }
 
+  /** Represents parameters for StartCall. */
```

Comment:
>StartCall [](http://example.com/codeflow?start=32&length=9)

Should this be using link syntax?

Reply:
Updated in latest commit

Reply:
I'm not that familiar with the `@link` tag but I think the part after the pipe symbol shouldn't be necessary since `startCall` is sufficiently descriptive and uses the correct casing (whereas StartCall has a capital at the beginning). Does this work?
```
 {@link startCall}
```